internal/types/account.go

```diff
@@ -1,3 +1,6 @@
+// Package types is an internal package that contains the types used in the Aptos Go SDK.
```

Comment:
I think the convention is for this kind of note to go in `doc.go` (with no other code content)

Reply:
All the doc guides, don't give enough conventions, just worry about syntax.  Will look into this